### PR TITLE
add gbi workshops

### PR DESCRIPTION
**Pull request by jburel**
Add gbi workshop

### CODE REVIEW

File: events/workshops-gbi-eosc-january-2021.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: news-events
+filename: events
+title: January 2021 OME users workshops
+main-blurb: 3-day online course (2.5 hours per day including a short break) organized by Global BioImaging and EOSC-Life.
+---          
+     <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Organisers: <a href="https://www.janelia.org/you-janelia/conferences/from-images-to-knowledge-with-imagej-friends" target="_blank">Global BioImaging and EOSC-Life</a></h2>
```
Comment (pwalczysko) [inline events/workshops-gbi-eosc-january-2021.html:10]: this  points to a wrong organizer

Comment (pwalczysko) [inline events/workshops-gbi-eosc-january-2021.html:10]:
```suggestion
                <h2 class="event-day">Organisers: <a href="https://www.globalbioimaging.org/international-training-courses-for-core-facility-staff/image-data-course" target="_blank">Global BioImaging and EOSC-Life</a></h2>
```

File: events/workshops-gbi-eosc-january-2021.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Dates: 27-29 January 2021</h2>
+            </div>
+        </div>
+        <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Duration: 4-hour workshops</h2>
```
Comment (pwalczysko) [inline events/workshops-gbi-eosc-january-2021.html:31]:
```suggestion
                <h2 class="event-day">2.5-hour for each workshops</h2>
```

File: events/workshops-gbi-eosc-january-2021.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+layout: news-events
+filename: events
+title: January 2021 OME users workshops
+main-blurb: 3-day online course (2.5 hours per day including a short break) organized by Global BioImaging and EOSC-Life.
```
Comment (pwalczysko) [inline events/workshops-gbi-eosc-january-2021.html:5]:
```suggestion
main-blurb: 3-day online course series (2.5 hours per day including a short break) organized by Global BioImaging and EOSC-Life. There will be 3 couses given by the OME within these series. These OME courses are covering the day 1 (OMERO), day 2 (NGFF) and the beginning of day 3 (IDR).
```

File: events/workshops-gbi-eosc-january-2021.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+layout: news-events
+filename: events
+title: January 2021 OME users workshops
+main-blurb: 3-day online course series (2.5 hours per day including a short break) organized by Global BioImaging and EOSC-Life. There will be 3 couses given by the OME within these series. These OME courses are covering the day 1 (OMERO), day 2 (NGFF) and the beginning of day 3 (IDR).
```
Comment (pwalczysko) [inline events/workshops-gbi-eosc-january-2021.html:5]:
```suggestion
main-blurb: 3-day online course series (2.5 hours per day including a short break) organized by Global BioImaging and EOSC-Life. There will be 3 workshops given by the OME within these series on OMERO, NGFF and IDR.
```

File: events/workshops-gbi-eosc-january-2021.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Dates: 27-29 January 2021</h2>
+            </div>
+        </div>
+        <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">2.5-hour for each workshops</h2>
```
Comment (pwalczysko) [inline events/workshops-gbi-eosc-january-2021.html:31]:
```suggestion
                <h2 class="event-day">2.5-hour per day for 3 days </h2>
```

File: events/workshops-gbi-eosc-january-2021.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+layout: news-events
+filename: events
+title: January 2021 OME users workshops
+main-blurb: 3-day online course series (2.5 hours per day including a short break) organized by Global BioImaging and EOSC-Life. There will be 3 workshops given by the OME within these series on OMERO, NGFF and IDR.
```
Comment (pwalczysko) [inline events/workshops-gbi-eosc-january-2021.html:5]:
```suggestion
main-blurb: 3-day online course series (2.5 hours per day) organized by Global BioImaging and EOSC-Life. There will be 3 workshops given by the OME within these series on OMERO, NGFF and IDR.
```

File: events/workshops-gbi-eosc-january-2021.html [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Dates: 27-29 January 2021</h2>
+            </div>
+        </div>
+        <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">2.5-hour per day for 3 days </h2>
```
Comment (pwalczysko) [inline events/workshops-gbi-eosc-january-2021.html:31]:
```suggestion
                <h2 class="event-day">Duration: 2.5-hour per day for 3 days </h2>
```

**Comment by pwalczysko** [comment]
Sorry for the edits, happy now with the formulations.
Built on https://snoopycrimecop.github.io/www.openmicroscopy.org/events/workshops-gbi-eosc-january-2021.html

cc @joshmoore @jburel

**Comment by pwalczysko** [comment]
Github actions seem to be unhappy with https://github.com/ome/www.openmicroscopy.org/commit/7de41908dff91701003d453d9926929173e22159#annotation_656015972

No idea why.

**Comment by jburel** [comment]
@pwalczysko I restarted the action and it is back to green
so all good

**Review by pwalczysko — APPROVED** [review]

File: events/index.html [hunk 8]
```diff
@@ -9,6 +9,12 @@
         <!-- begin Events -->
         <hr class="whitespace">
         <div id="community-posts" class="row">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> 27-29 January, 2021</h6>
+                <h2><a href="workshops-gbi-eosc-january-2021.html">OME Workshops, virtual course</a></h2>
```
Comment (joshmoore) [inline events/index.html:14]: Thoughts on adding "GBI/EOSC" here?

File: events/workshops-gbi-eosc-january-2021.html [hunk 9]
```diff
@@ -0,0 +1,35 @@
+---
+layout: news-events
+filename: events
+title: January 2021 OME users workshops
+main-blurb: 3-day online course series (2.5 hours per day) organized by Global BioImaging and EOSC-Life. There will be 3 workshops given by the OME within these series on OMERO, NGFF and IDR.
```
Comment (joshmoore) [inline events/workshops-gbi-eosc-january-2021.html:5]: within "this" series.

**Review by joshmoore — COMMENTED** [review]
Added minor thoughts, but nothing urgent. Was there discussion about the deployment of this?

**Comment by pwalczysko** [comment]
> Added minor thoughts, but nothing urgent. Was there discussion about the deployment of this?

No, my slightly lazy thought was that now with the new workflow it will magically happen, which is not true probably.  :)
Did the deployment workflow change by introducing github actions (it certainly did at the back end, but is the result achieved by the same steps as with travis or does something need to be changed ?)  cc @sbesson

File: events/index.html (hunk 8 above)
Comment (pwalczysko) [reply events/index.html:14]: I am all for it, maybe

``OME workshops run within a GBI.EOSC-Life course``

? cc @jburel

Comment (jburel) [reply events/index.html:14]: make sense

**Comment by joshmoore** [comment]
The script that Simon prepared is available on ome-www with a few modifications from me. I can use it or someone else can try it out: 

https://github.com/ome/www.openmicroscopy.org/pull/426

**Comment by pwalczysko** [comment]
> The script that Simon prepared is available on ome-www with a few modifications from me. I can use it or someone else can try it out:

Trying it out straight on a production server makes me uncomfortable - is there a way to run it against ``ome-www-dev`` first ? I am also not sure from where to execute:
1. inside the ``ome-www`` server to execute ? 
2. Or locally within the checkout of this www... repo ?

**Comment by joshmoore** [comment]
If you have my additions then it will do a dry-run. It should run from anywhere.

**Comment by sbesson** [comment]
>  Did the deployment workflow change by introducing github actions (it certainly did at the back end, but is the result achieved by the same steps as with travis or does something need to be changed ?) cc @sbesson

The continuous integration service was changed but in terms of process nothing should have changed as the result of the switch to GitHub actions. PRs are still tested individually and release artifacts that can be used for deployment are created on tags.

As mentioned above, https://github.com/ome/www.openmicroscopy.org/pull/426 aimed at simplifying the deployment of these artifacts.

**Comment by pwalczysko** [comment]
> If you have my additions then it will do a dry-run. It should run from anywhere.

Not comfortable about the setup and testing levels - still having the feeling that we are about to run a completely untested script onto a live production system, see https://github.com/ome/www.openmicroscopy.org/pull/426#issuecomment-739854299 and https://github.com/ome/www.openmicroscopy.org/pull/426#issuecomment-739857592.

What advantages does the script actually have over the playbook ?

**Comment by joshmoore** [comment]
> still having the feeling that we are about to run a completely untested script onto a live production system

https://github.com/ome/www.openmicroscopy.org/pull/426#issuecomment-734277376 ?

I'm not sure what you're looking for.

> What advantages does the script actually have over the playbook ?

Speed, transparency, and the ability to be run as a cron job so that we don't need to run anything.

**Pull request merged**